contracts/SnappBase.sol

```diff
@@ -151,7 +149,10 @@ contract SnappBase is Ownable {
             "Unsuccessful transfer"
         );
 
-        if (deposits[depositIndex].size == DEPOSIT_BATCH_SIZE || block.number > deposits[depositIndex].creationBlock + 20) {
+        if (withdrawIndex == MAX_UINT ||
```

Comment:
If solidity short circuits, would it be long-term more efficient to include this condition at the end of the statement?

Reply:
Yes, however it would only accelerate a single order every 3 minutes by a tiny amount of gas.